net/ipv4/tcp_output.c

```diff
@@ -2202,7 +2202,6 @@ int tso_fragment(struct sock *sk, struct sk_buff *skb, unsigned int len,
 
 	return 0;
 }
-EXPORT_SYMBOL(tso_fragment);
```

Comment:
We also added the function to `include/net/tcp.h`, so now we can remove it

Reply:
Yes I remove it from tcp.h